languages/elixir/exercises/concept/maps/.docs/hints.md

```diff
@@ -0,0 +1,60 @@
+### General
+
+- A [map][maps] are a associative data structure of key-value pairs.
+- Elixir offers [many useful Map module functions in the standard library][map-module].
+
+### 1. Define a new high score map
+
+- You need to define a [named function][named-function] with 0 parameters.
+- It should return an empty [map][maps].
+
+### 2. Add players to the high score map
+
+- You need to define a [named function][named-function] with 3 parameters.
+  - The first argument is a [map][map] of scores.
+  - The second argument is a name [string][strings].
+  - The third argument is the score [integer value][integers], it is optional, so a [default value][default-arg] should be specified.
+- The resulting map should be returned
+
+### 3. Remove players from the score map
+
+- You need to define a [named function][named-function] with 2 parameters.
+  - The first argument is a [map][map] of scores.
+  - The second argument is a name [string][strings].
```

Comment:
Is there a dependency that ensure students know how to work with strings?

Reply:
yes, there is a string-literal concept earned in the lists exercise to show the basic string type as data.

We are purposely not manipulating strings until after the pattern matching exercises

Reply:
Okay! Just checking :)